plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSource.java

```diff
@@ -112,8 +114,14 @@ else if (column.getName().equals(ROW_ID_COLUMN_NAME)) {
                 delegateIndex++;
             }
             else {
```

Comment:
Nit, if we're going to leave this in let's inline this if/else with the main one
```
else if (nullColumnNames.contains(...)) {
...
}
else {
...
}
```